### ISSUE TITLE
Ctrl + Space for multiselect doesn't work on Linux

### ISSUE THREAD

**Issue by WofWca** [deltachat/deltachat-desktop#6209]
- Operating System (Linux/Mac/Windows/iOS/Android): Linux
- Delta Chat Version: 2.48.0
- Expected behavior: Ctrl + Space toggles selection of chat list items
- Actual behavior:
- Steps to reproduce the problem:
- Screenshots:
- Logs:

https://github.com/deltachat/deltachat-desktop/blob/a4cbe0806328a79af1135ae464486c57bf40af48/packages/frontend/src/hooks/useMultiselect.ts#L233-L258

On Windows it works, but not on Linux.

I believe we have to also be listening for `kwydown`. But we can't remove `onClick` listener because we need it for mouse clicks. Then I guess we have to introduce some logic to only run the function once even if both of the events fired.

**Comment by WofWca** [comment]
Oh, and I think adding the onkeydown event handler will make us better prepared for adding multi-select to the message list. Because in the message list, the selectable elements are not buttons. They don't receive the on-click event when you press spacebar.

**Comment by nicodh** [comment]
Ctrl + Space hast no keybinding ?? What do you expect?

**Comment by WofWca** [comment]
> Ctrl + Space hast no keybinding ?? 

Not sure what you mean?

> What do you expect?

As I wrote, Ctrl + Space should select / unselect the currently focused chat. Roughly as Shift + Space, but not contiguously.

But I just realized that we have tests for this. Maybe this issue is only reproducible in Electron? Or only on my OS?

https://github.com/deltachat/deltachat-desktop/blob/c343bff00c53553d2036cf21eca0cc47af74bc16/packages/e2e-tests/tests/chat-list-multiselect.spec.ts#L114-L129

**Comment by WofWca** [comment]
> Maybe this issue is only reproducible in Electron? Or only on my OS?

Ahhh I see. It's because the tests use `ControlOrMeta`. Indeed on Linux it's also possible to select with "Windows (a.k.a. Meta) + Space". But Ctrl should also work I think.

**Comment by WofWca** [comment]
Hmmmm I tried to change `ControlOrMeta` to `Control` in https://github.com/deltachat/deltachat-desktop/pull/6247, but the tests still pass, both in CI and on my Linux system, with `pnpm e2e`.
Also Playwright docs on https://playwright.dev/docs/api/class-keyboard#keyboard-press say:
> ControlOrMeta resolves to Control on Windows and Linux and to Meta on macOS.

But on the actual dev version of Delta Chat Ctrl + Space doesn't work. I tried KDE and Gnome, Wayland.

Edit: anyways, as I said, we'll probably need to have the `onKeyDown` handler either way, to support non-`<button>` target elements.

**Comment by nicodh** [comment]
This is more a missing feature than a bug

**Comment by WofWca** [comment]
Well the feature exists and works on Windows and in E2E tests, so IDK.
